clients/drcachesim/common/options.cpp

```diff
@@ -505,6 +505,14 @@ droption_t<bytesize_t>
                  "of being simulated.  This skipping may be slow for large skip values; "
                  "consider -skip_instrs for a faster method of skipping.");
 
+droption_t<bytesize_t> op_L0_filter_until_instrs(
+    DROPTION_SCOPE_CLIENT, "L0_filter_until_instrs", 0,
+    "Number of instructions for warmup trace",
+    "Specifies the number of instructions to run in warmup mode. In warmup mode, we "
+    "filter accesses through the L0{D,I}_filter caches. The size of these can be "
+    "specified using L0{D,I}_size. The filter instructions come after the "
+    "-trace_after_instrs count and before the instruction trace. ");
```

Comment:
The use of windows has implications the user must be aware of: interactions with -split_windows and -retrace_every_instrs, e.g.  This should be documented.

Reply:
Added documentation.